shacl/BrickEntityShapeBase.ttl

```diff
@@ -96,6 +102,23 @@ bsh:PointShape a sh:NodeShape;
     ];
     .
 
+bsh:SystemShape a sh:NodeShape;
+    sh:targetClass brick:System;
+    sh:and (
+        [sh:not [ sh:class brick:Equipment ] ; sh:message "System is an exclusive top class." ]
+        [sh:not [ sh:class brick:Location ] ; sh:message "System is an exclusive top class." ]
+        [sh:not [ sh:class brick:Substance ] ; sh:message "System is an exclusive top class." ]
+        [sh:not [ sh:class brick:Quantity ] ; sh:message "System is an exclusive top class." ]
+        [sh:not [ sh:class brick:Point ] ; sh:message "System is an exclusive top class." ]
+    );
+    sh:message "System is an exclusive top class." ;
+    sh:property [
+        sh:path brick:hasPart;
+        sh:class brick:Equipment;
+        sh:message "A System can be associated with Equipments."
+    ];
```

Comment:
Systems can have other Systems, Equipment, Locations and Points as parts

Reply:
At least in the proposed #269 ; let me know if you have a different opinion! I've written up some basic docs on the idea here: https://github.com/BrickSchema/docs/pull/3/files